CHANGELOG.rst

```diff
@@ -9,10 +9,15 @@ UNRELEASED - Under development
 [2024.1.0-b1] - 2024-08-05
 ***********************
 
+Added
+=====
+- Added new "k-modal" component.
```

Comment:
Heriberto, can you add this under the `UNRELEASED` section? This change here will be for the next major release. If we had unit tests already in place I'd be inclined to try to include in the `2024.1` but since the UI requires some extra manual effort to fully validate then let's leave for the next release.